run_happy.inputs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "##_COMMENT1": "Input/Output Files", 
+  "HappyWorkflow.vcf_fofn": "File",
+
+  "##_COMMENT1": "References/Resources",
+  "HappyWorkflow.reference_fasta_index": "File",
+  "HappyWorkflow.reference_fasta": "File",
+  "HappyWorkflow.reference_dict": "File",
+  "HappyWorkflow.truth_vcf": "File",
+  "HappyWorkflow.truth_vcf_index": "File",
+  "HappyWorkflow.truth_bed": "File",
+  "HappyWorkflow.rscript": "File",
```

Comment:
We should provide the rscript in the workspace too.  It's next to the WDL in the gatk repo.

Reply:
called `happy_plot.R`